packages/cms/src/components/Viz/PercentageBar.css

```diff
@@ -41,3 +42,88 @@
   color: var(--stat-value-color);
   margin-bottom: 0;
 }
+/* Added styling */
+.percentage-bar-list{
+  padding-bottom: 1rem;
+}
+.percentage-bar-list > li {
+  display: flex;
+  flex-direction: row;
+}
+.percentage-bar-item .label{
+  width: 20%;
+}
+.percentage-bar-label {
+  margin-bottom: 0.25em;
+  color: var(--stat-label-color);
+  width: 24%;
+}
+
+.percentage-bar-value {
+  color: var(--stat-value-color);
+  margin-bottom: 0;
+  width: 6%;
+}
+
+.percentage-bar-item{
+  font-size: .75rem;
+}
+.percentage-bar-bg,
+.percentage-bar{
+  border-radius: 0;
+  height: 1rem;
+  width: 74%;
+}
+.percentage-bar-bg{
+  background-color: transparent;
+}
+
+
+/* PrecentageBar Grid */
+.percentage-bar-wrapper{
+    position: relative;
+    margin-bottom: 1rem;
+}
+.precentage-bar-grid-holder{
+    width: 74%;
+    margin-left: 26%;
+    height: 100%;
+    position: absolute;
+}
+.percentage-bar-grid{
+    position: absolute;
+    width: 100%;
+    height:100%;
+}
+.percentage-bar-ticks{
+    clip-path: url(#percentage-bar-clip);
+}
+.percentage-bar-ticks line{
+    stroke: #e5e5e5;
+    stroke-width: 2;
```

Comment:
Please replace with `stroke-width: var(--percentage-bar-stroke-width)` and [add a new line to style.yml](https://github.com/Datawheel/canon/blob/master/packages/cms/app/style.yml#L209): `percentage-bar-stroke-width: "2px"`